static/js/recent_topics_util.js

```diff
@@ -16,6 +16,7 @@ export function is_in_focus() {
         !compose_state.composing() &&
         !popovers.any_active() &&
         !overlays.is_active() &&
+        !overlays.is_modal_open &&
```

Comment:
You forgot to call the function, I think!

Reply:
Oops!